lib/train/transports/gcp.rb

```diff
@@ -88,6 +88,8 @@ def connect
         scopes = ['https://www.googleapis.com/auth/cloud-platform',
                   'https://www.googleapis.com/auth/compute']
         authorization = Google::Auth.get_application_default(scopes)
+        Google::Apis::ClientOptions.default.application_name = 'chef-train'
```

Comment:
Should we use inspec here, since users may not know train. Alternately I propose to use `chef-inspec-train`

Reply:
@chris-rock I did struggle with exactly what name should be given.  Is train used by anything other than InSpec at the moment?  Will it be in the future? 

It might be better if this was something that InSpec, or other libraries, could pass to train when making connections.  

Either way, I would be happy with this:

```suggestion
        Google::Apis::ClientOptions.default.application_name = 'chef-inspec-train'
```

Reply:
I am also not 100% sure. other Chef products use train, but not the cloud plugins. I recommend we go with `chef-inspec-train` for now and make it customizable via a plugin option for the using application going forward

Reply:
pushed an update with this new `application_name`